tests/cts/test_sparql_resolver/test_sqlalchemy.py

```diff
@@ -2,8 +2,9 @@
 from capitains_nautilus.cts.resolver import SparqlAlchemyNautilusCTSResolver
```

Comment:
`SparqlAlchemyNautilusCTSResolver` should probably be `SparqlAlchemyNautilusCtsResolver`